internal/core/src/index/knowhere/knowhere/common/Utils.cpp

```diff
@@ -78,6 +78,15 @@ Assemble(BinarySet& binarySet) {
 void
 Disassemble(const int64_t& slice_size_in_byte, BinarySet& binarySet) {
     milvus::json meta_info;
+    auto slice_meta = binarySet.Erase(INDEX_FILE_SLICE_META);
+    if (slice_meta != nullptr) {
+        milvus::json last_meta_data =
+            milvus::json::parse(std::string(reinterpret_cast<char*>(slice_meta->data.get()), slice_meta->size));
+        for (auto& item : last_meta_data[META]) {
```

Comment:
What if the item is duplicated with the later slice_i?

Reply:
It's impossible, size of data in last_meta_data has been smaller than slice_size_in_byte, and will not be duplicated with the later slice_i.